packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -264,6 +329,199 @@ contract LockedGold is
     return totalNonvoting;
   }
 
+  /**
+   * Delegates CELO to delegatee.
+   * @param delegatee The delegatee account.
+   * @param percentageToDelegate Percents of total CELO that will be delegated from delegatee.
+   */
+  function delegateGovernanceVotes(address delegatee, uint256 percentageToDelegate) external {
+    require(
+      percentageToDelegate > 0 && percentageToDelegate <= 100,
+      "delegated percents can be only between 1%..100%"
+    );
+    address delegatorAddress = getAccounts().voteSignerToAccount(msg.sender);
+    address delegateeAccount = getAccounts().voteSignerToAccount(delegatee);
+
+    Delegated storage delegated = delegatorInfo[delegatorAddress];
+    delegated.delegatees.add(delegateeAccount);
+    require(delegated.delegatees.length() <= maxDelegateesCount, "Too many delegatees");
+
+    DelegatedInfo storage currentDelegateeInfo = delegated
+      .delegateesWithPercentagesAndAmount[delegateeAccount];
+
+    require(
+      percentageToDelegate >= currentDelegateeInfo.percentage,
+      "Use revokeDelegatedGovernanceVotes to decrease delegated amount."
```

Comment:
The message should also include the reason why it failed, although giving a suggestion on how to fix it is great.
(This would also answer my previous question)